README.md

```diff
@@ -1 +1,108 @@
 
+## What Should You Know
+The knowledge that you should still know how to use Git commands in the command line. While using git in VS code is very convenient, you will eventually need to use the command line when moving away from VS Code.
+
+## Setting Up Your Dev Branch
+
+### Step 1
+Open VS Code. Locate your local repo. If you have already opened the repo before, with or without using a Dev Container, you can find it in a list of 5 items below the section labeled recently opened. If you re-opened a Dev Container repo you opened before, skip to Step 5.
+
+<img src = "PicsForGuide/1 OpenVSC.png" href = "VS Code Home Page"/>
+
+### Step 2
+After opening a repo, look at the bottom left of the VS code and click on the icon that looks like the image below.
+
+<img src= "PicsForGuide/3 howVSCLookswithnoDevP2.png" href = "Dev Container icon with no container"/>
+
+### Step 3
+After clicking the icon, a drop-down menu should open in the top middle of the screen. Once you locate the menu, click re-open in Dev Container. (As a side note, ensure that Docker is running in the background. If it's not running, the Dev Container will not open.)
+
+<img src= "PicsForGuide/4 reOpenDC.png" href= "Dev Container drop down menu"/>
+
+### Step 4
+To ensure the Dev Container is running properly, look back at the bottom left corner of the VS code where the icon was. It should look like two arrowheads sliding past each other, as seen in the picture below.
+
+<img src = "PicsForGuide/5 IfYouAreinDCbottom.png" href = "Dev Container after Connecting" width = 300px />
+
+### Step 5
+After checking that you are in the Dev Container and locating the file you want to change, you can start looking for the Source Control icon, as seen below.
+
+<img src = "PicsForGuide/6 IfYouAreinDCOpenSC.png" href = "Source Control icon" width = 300px />
+
+### Step 6
+On top of ensuring you locate Source Control, you also need to make sure you are on the correct branch, or if you don't have a branch, how you could make one. To tell if you are on the correct branch, you can navigate back down to the bottom left of the VS code and look right next to the blue Dev container information. If it says main, you are in the wrong branch.
+
+<img src = "PicsForGuide/7 CheckBranch.png" href = "Main branch display"/>
+
+#### Below is the image guide with some text for making a new branch
+1. Click on where it displays the current branch name in the bottom left. Afterward, another drop-down menu will open, and you will click Create a new branch. If you want to change branches, you can select existing branches from the same drop-down menu.
+
+<img src= "PicsForGuide/8 MakeNewBranch.png" href = "Drop down for making new branches"/>
+
+2. Next, you would put in a branch name and hit enter, as shown below.
```

Comment:
"Next, put in a branch name and hit enter, as shown below"

Use present active tense throughout and minimize passive voice.